packages/gatsby-image/src/index.js

```diff
@@ -100,11 +100,34 @@ const noscriptImg = props => {
     ? `crossorigin="${props.crossOrigin}" `
     : ``
 
-  return `<picture>${srcSetWebp}<img ${width}${height}${sizes}${srcSet}${src}${alt}${title}${crossOrigin}style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/></picture>`
+  // Since we're in the noscript block for this image (which is rendered during SSR or when js is disabled),
+  // we have no way to "detect" if native lazy loading is supported by the user's browser (at least not today — See 1)
+  // Since this attribute is a progressive enhancement, it won't break a browser with no support
+  // Therefore setting it by default is a good idea.
+
+  // 1. In the future, a Client Hint for this might be available (https://bugs.chromium.org/p/chromium/issues/detail?id=949365)
+
+  const loading = props.loading ? `loading="${props.loading}" ` : ``
```

Comment:
we should default `loading` to lazy since that's our existing behavior

Reply:
Oh, looks like below we do default to `lazy`? In that case, we don't need to check if the prop is set.

Reply:
If the attribute isn't set, and a browser supports native lazy loading, they default to `auto` which will use `eager` or `lazy` strategies based on distance from viewport afaik?

We can also drop this noscript string approach if min version of React for Gatsby can be 16.5+ which has fixed the issue that required changing to this approach for noscript support.